apps/judicial-system/web/src/components/IndictmentCaseFilesList/IndictmentCaseFilesList.tsx

```diff
@@ -48,6 +48,7 @@ interface RenderFilesProps {
   caseFiles: CaseFile[]
   onOpenFile: (fileId: string) => void
   showFiledDocumentNumber?: boolean
+  showUserGeneratedFileName?: boolean
```

Comment:
Is there any reason not to show the user generated file name always? If not, then we could simplify this component. Consider discussing this with Guðný and make the change if she agrees.

Reply:
Synced with Guðný and it makes sense to rather use the user generated name if it exists, otherwise the original file name 👍